validator/db/kv/attestation_history_new.go

```diff
@@ -198,3 +198,51 @@ func (store *Store) ImportOldAttestationFormat(ctx context.Context) error {
 	err = store.SaveAttestationHistoryNewForPubKeys(ctx, dataMap)
 	return err
 }
+
+// UpdateAttestationProtectionDb exports old attestation protection data
+// format to the new format and save the exported flag to database.
+func (store *Store) UpdateAttestationProtectionDb(ctx context.Context) error {
+	importAttestations, err := store.shouldImportAttestations()
+	if err != nil {
+		return err
```

Comment:
We should aim to use better formatted errors to diagnose failures at runtime, so perhaps errors.Wrap or fmt.Errorf with a better description for some of these

Reply:
added many erros.Wrap